### PR TITLE
add custom url schemes for Bowser

### PR DESCRIPTION
**Pull request by paericksen**
resolves issue #26 

added custom url schemes `bowser://` for `http` and `bowsers://` for `https`

### CODE REVIEW

File: bowser-ios/Bowser.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -4024,6 +4024,11 @@
 				CLASSPREFIX = Bowser;
 				LastUpgradeCheck = 0600;
 				ORGANIZATIONNAME = "Ericsson Research";
+				TargetAttributes = {
+					B7E3D0D015625681003DE0F1 = {
+						DevelopmentTeam = 8J7L2VBRVP;
```
Comment (stefanalund) [inline bowser-ios/Bowser.xcodeproj/project.pbxproj]: This is your iOS Dev team ID. Please select "None" before pushing the code.

File: bowser-ios/Bowser/BowserAppDelegate.m [hunk 2]
```diff
@@ -35,6 +35,10 @@
 
 @implementation BowserAppDelegate
 
+- (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:(NSDictionary *)launchOptions{
```
Comment (stefanalund) [inline bowser-ios/Bowser/BowserAppDelegate.m]: For readability, use the convention with { in methods on a new line.

File: bowser-ios/Bowser/BowserAppDelegate.m [hunk 3]
```diff
@@ -64,7 +68,18 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     // Override point for customization after application launch.
     return YES;
 }
-							
+
+- (BOOL)application:(UIApplication *)application openURL:(NSURL *)url sourceApplication:(NSString *)sourceApplication annotation:(id)annotation {
```
Comment (stefanalund) [inline bowser-ios/Bowser/BowserAppDelegate.m]: Same comment as before, { on a new line.

File: bowser-ios/Bowser/BowserViewController.m [hunk 4]
```diff
@@ -163,6 +171,18 @@ - (void)viewDidAppear:(BOOL)animated
     isBridgeInitialized = YES;
 }
 
+- (void)observeValueForKeyPath:(NSString *)keyPath ofObject:(id)object change:(NSDictionary *)change context:(void *)context{
```
Comment (stefanalund) [inline bowser-ios/Bowser/BowserViewController.m]: New line.

**Comment by stefanalund** [comment]
Nice works, just some minor style fixes and then it's ready to merge.

**Comment by stefanalund** [comment]
You need to rebase this before I can land it.

**Comment by stefanalund** [comment]
@paericksen do you mind rebasing this to latest master so that I can land it? Thanks.

**Comment by paericksen** [comment]
rebase failed due to merge conflicts, I'll work on resolving the conflicts when I have a few spare minutes (hopefully this week).

**Comment by stefanalund** [comment]
OK thanks. Should probably not be a big problem since not much has changed.

**Comment by HBIOSDev** [comment]
Is this change uploaded to App Store version of Bowser ? 
I have checked appstore version and it is not working.  
I tried bowser://www.xyzsite.com but this didnt worked. 
If that is not uploaded, when this feature is going to get live ?

**Comment by superdump** [comment]
We'll be making a new submission to the App Store soon.

**Pull request merged**